### DISCUSSION TITLE
How to add apps to the project

### DISCUSSION THREAD

**Discussion by Soleil-des-chats** [DustinBrett/daedalOS#430] [Q&A]
Hello, i forked the project and i would like to know how to add my own apps to it, thanks in advance.

**Comment by DustinBrett** [comment]
There isn't yet an official way, but if you have the source code then there are quite a few examples you could follow. The apps are in [components/apps](https://github.com/DustinBrett/daedalOS/tree/main/components/apps) and there are a few different kinds. There are ones like FileExplorer or Messenger where they have been fully custom made. There is also ones like Paint or IRC where it just uses an iframe basically. And there are the ones which use a canvas like Quake3 & JSDOS. I also have ones like Webamp where they have no Window container of their own. It will depend what kind of app you want to make for which to use, but there are many examples to go from.

**Reply by Soleil-des-chats** [reply]
Thx for the answer, i’m going to try and see if it work

**Reply by DustinBrett** [reply]
Cool. If you have specific questions or code of the app existing elsewhere, I can suggest further steps.
